docs/infra/api-markdown-documenter/admonition-node.mjs

```diff
@@ -48,15 +50,57 @@ export const admonitionNodeType = "Admonition";
 export class AdmonitionNode extends DocumentationParentNodeBase {
 	/**
 	 * @param {BlockContent[]} children - Child node content.
-	 * @param {string} admonitionKind - The kind of admonition. See {@link https://docusaurus.io/docs/markdown-features/admonitions}.
+	 * @param {AdmonitionKind} admonitionKind - The kind of admonition. See {@link https://docusaurus.io/docs/markdown-features/admonitions}.
 	 * @param {string | undefined} title - (Optional) Title text for the admonition.
 	 */
 	constructor(children, admonitionKind, title) {
 		super(children);
 
-		this.type = admonitionNodeType;
+		this.type = "admonition";
```

Comment:
`Admonition -> admonition` is expected? Maybe similar to the `Warning -> warning` in another file?